README.md

```diff
@@ -467,27 +467,24 @@ form +++
                 $0.placeholder = "Tag Name"
             }
         }
-        $0 <<< NameRow() {
```

Comment:
This is the default row, otherwise the user to enter the first name will have to tap add button to insert a NameRow.